pom.xml

```diff
@@ -12,6 +12,8 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <scala.version>2.13.3</scala.version>
         <scala.version.short>2.13</scala.version.short>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```

Comment:
It seems that setting `maven.compiler.release` is an even better option (see https://stackoverflow.com/questions/59049980/maven-compiler-release-as-an-replacement-for-source-and-target)